robot_skills/src/robot_skills/util/robot_constructor.py

```diff
@@ -1,5 +1,10 @@
-def robot_constructor(robot_name):
+import sys
+
+def robot_constructor(robot_name=None):
     """Construct a robot by it's name. Choices are amigo, sergio, mockbot"""
+    if not robot_name:
+        robot_name = sys.argv[1]
```

Comment:
This way, you can just call 

```python
robot = robot_constructor()
```

instead of the slightly more lines of code:
```python
import sys
robot_name = sys.argv[1]
robot = robot_constructor(robot_name)
```

But I agree the advantage is very small.